### DISCUSSION TITLE
Could we load a log file from local disk instead of org?

### DISCUSSION THREAD

**Discussion by kjonescertinia** [certinia/debug-log-analyzer#14] [General] [answered]
[From Issue #13]

I don't have my IDE connected to orgs like uat and regression so it'd be nice if I could download/analyze a local file in the project folder. Also cloud logs are somewhat ephemeral so I might have use to download the log if it's an issue I might be looking at for more than one day.

**Comment by kjonescertinia** [comment, marked as answer]
That should just work fine, in fact it's also how I normally work. To open the the downloaded file you want to make sure the name ends with ".log" & then either right-click on the name in the file view or open the file and right-click. On the menu you should see 'Log: Show Log Analysis' that will open the log analysis window for you.

Please let us know if this does not work for you. There may be some other reason you can't open it.

**Reply by lcottercertinia** [reply]
As well as the command palette, you can right click in the file in the explorer or directly in the file view and coming soon there will be a code lens at the top of a log file when it is opened.
